src/containers/SubjectPage/SubjectContainer.jsx

```diff
@@ -63,19 +62,11 @@ const SubjectPage = ({
 
   const { editorsChoices, layout, about, metaDescription } = subjectpage;
 
-  const [topic, setTopic] = useState(null);
-  const [subTopic, setSubTopic] = useState(null);
-  const [subSubTopic, setSubSubTopic] = useState(null);
-  const [currentLevel, setCurrentLevel] = useState('Subject');
+  const [currentLevel, setCurrentLevel] = useState(0);
+  const [breadCrumbList, setBreadCrumbList] = useState([]);
 
   useEffect(() => {
```

Comment:
Denne useEffecten som kjøres ved rendering gjør at willTrackPageview kjører to ganger og dermed sender to pageview til google. Kan pageviews flyttes til SubjectPageContent?

Reply:
Det skal jeg få ordnet 👍

Reply:
Og så testene sjølvsagt.

Reply:
Mulig du må sende inn loading fra SubjectPage for å forhindre ekstra pageviews.

Reply:
Det ser ut til at den fyrer av tracking i det man klikker på en emne knapp og så en ny en når innholdet er lastet inn. Prøver å sende opp loading fra Topic, men har ikke helt fått det til å funke.

Reply:
Heile denne effecten er for å legge på filter dersom det ikkje er satt og dersom det finnes på det laveste nivået. Den gjør uansett en history.replace og dermed trigges to pageviews. Er løsninga så enkel at vi ikkje gjør replace om vi ikkje trenger det?

Reply:
Tror ikke history.replace effecten trigges når man velger emner nedover i struktur, men tracking trigges dobbelt der. Så jeg tror desverre ikke det fikser problemet.

Reply:
Utkast til useEffect som kun reloader dersom filter ikkje er satt i url og det finnes flter på emnet.
```
  useEffect(() => {
    const lowermostId = topics[topics.length - 1];
    const lowermost =
      subject.allTopics.find(topic => topic.id === lowermostId) || subject;
    const subjectFilters = lowermost?.filters?.filter(f =>
      subject.filters?.map(f2 => f2.id).includes(f.id),
    );
    const selectedFilter = subjectFilters?.[0]?.id;
    const filters = activeFilterId || selectedFilter;
    const filterParam = filters ? `?filters=${filters}` : '';
    const path = parseAndMatchUrl(location.pathname, true);
    const shouldReload = !activeFilterId && selectedFilter;
    if(shouldReload) {
      if (path) {
        history.replace({ pathname: path.url, search: filterParam });
      } else {
        // no topics in path
        history.replace({
          pathname: toSubjects() + subject.path,
          search: filterParam,
        });
      }
    }
  }, []);
```